app/auth/views.py

```diff
@@ -0,0 +1,343 @@
+from . import auth_blueprint
+from flask.views import MethodView
+from flask import make_response, request, jsonify, abort
+
+from app.models import User, BucketList, BucketlistItems
+
+
+class RegistrationView(MethodView):
+    """Create a new user"""
+
+    def post(self):
+        """handle post requests for this view. URL: /auth.register/"""
+
+        user = User.query.filter_by(email=request.data['email']).first()
```

Comment:
i think it'd be better to set and validate the data in the request before this query. (in this case the username and password too).
also prefer `.get()` to avoid `bad request` errors if any of them is missing in the request